File: app/src/main/java/com/example/jdgomes/desfrute/formulario/GastosFormulario.java
package com.example.jdgomes.desfrute.formulario;


public class GastosFormulario {

    private String[] itensDespesaMock;


    public GastosFormulario() {
        this.loadData();
    }


    private void loadData() {

        this.setItensDespesaMock(new String[] {
                "Itens 1", "Itens 2", "Itens 3", "Itens 4"
        });
    }


    public String[] getItensDespesaMock() {
        return itensDespesaMock;
    }

    public void setItensDespesaMock(String[] itensDespesaMock) {
        this.itensDespesaMock = itensDespesaMock;
    }
}

File: app/src/main/java/com/example/jdgomes/desfrute/formulario/GastosFormulario.kt
package com.example.jdgomes.desfrute.formulario


class GastosFormulario {

    var itensDespesaMock: Array<String>? = null

    init {
        this.loadData()
    }


    private fun loadData() {

        this.itensDespesaMock = arrayOf("Itens 1", "Itens 2", "Itens 3", "Itens 4")
    }
}
